helm/prometheus-operator-app/values.yaml

```diff
@@ -10,11 +10,17 @@ prometheus-operator-app:
     ingressPerReplica:
       ingressClassName: nginx
       pathType: ImplementationSpecific
+  coreDns:
+    service:
+      selector:
+        k8s-app: coredns
   grafana:
     ingress:
       ingressClassName: nginx
   kubeStateMetrics:
     enabled: false
+  nodeExporter:
```

Comment:
If you want to fix default enabled 3rdparties, nodeExporter is not the only one.
See https://github.com/giantswarm/config/pull/1332